tests/auth_tests/test_validators.py

```diff
@@ -284,7 +284,8 @@ def test_username_validators_help_text(self):
             {
                 'NAME': 'django.contrib.auth.validators.UsernameMinimumLengthValidator',
                 'OPTIONS': {
-                    'min_length': 3,
+                    'limit_value': 3,
+                    'message': 'This password is too short',
```

Comment:
username, not password. :-)

Reply:
> glad to see this moving along. some minor fixes after your last push.

Thank you! that is so embarrassing. I actually read everything in the docs this time in html.

-- Fixed all the typos and the line break.
-- Added the new method help_text for Unicode and ASCII username validator in the docs
-- Added the UsernameMinimumLength validator in list of Validators as well.